package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Simple Firebase Server",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run cover && npm run test-es5",
-    "cover": "istanbul cover ./node_modules/mocha/bin/_mocha -- -r test/mocha-babel.js test/*.spec.js",
-    "test-es5": "mocha test/es5-only-test.js",
-    "debug": "mocha -r test/mocha-babel.js test/*.spec.js",
+    "test": "npm run lint && npm run cover",
+    "cover": "istanbul cover ./node_modules/mocha/bin/_mocha",
+    "debug": "mocha",
+    "watch": "mocha -w",
```

Comment:
Using `mocha.opts` made all these scripts a lot simpler. Not everyone wants to install `mocha` globally, so the `debug` and `watch` scripts are still valuable.